### DISCUSSION TITLE
NeMo Guardrails

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#19] [General]
Discuss things related to NeMo Guardrails here!

**Comment by itsnotmik** [comment]
NeMo Guardrails installation can be found here: https://docs.nvidia.com/nemo/guardrails/getting_started/installation-guide.html

Requires:
* Python 3.8, 3.9, 3.10
* GCC and G++

**Comment by ACraig7** [comment]
NeMo Guardrails is [open source](https://github.com/NVIDIA/NeMo-Guardrails/blob/develop/LICENSE.md),
